### DISCUSSION TITLE
安卓相册无内容时，选择器顶部分类UI不正常

### DISCUSSION THREAD

**Discussion by DA-SHUIBI** [fluttercandies/flutter_wechat_assets_picker#504] [Q&A]
测试了安卓10和安卓13都有此类问题，flutter_wechat_assets_picker版本为8.7.1，请问有没有解决办法？

![image](https://github.com/fluttercandies/flutter_wechat_assets_picker/assets/59349501/185474af-b1e2-4e9b-8968-a554fa7405ae)

**Comment by AlexV525** [comment]
也并没有很不正常，正常的应该是什么状态？

**Reply by DA-SHUIBI** [reply]
当相册中完全没有资源，且相册中没有文件夹时，正常来说顶部切换类型的选择器应该消失或者展示无可选项

**Reply by AlexV525** [reply]
微信的做法是？

**Reply by DA-SHUIBI** [reply]
微信中没试过，但是顶部选择器无选项时，应该消失或显示无选项，而目前的这个状态的UI虽然无伤大雅，只是影响用户体验

**Reply by AlexV525** [reply]
https://github.com/fluttercandies/flutter_wechat_assets_picker/assets/15884415/7d582463-ec94-4a36-90e0-e178bf6d9ef7

你看看微信这叫用户体验好吗...
